client/package.json

```diff
@@ -13,52 +13,52 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~11.2.4",
-    "@angular/cdk": "~11.2.3",
-    "@angular/common": "~11.2.4",
-    "@angular/compiler": "~11.2.4",
-    "@angular/core": "~11.2.4",
-    "@angular/flex-layout": "~11.0.0-beta.33",
-    "@angular/forms": "~11.2.4",
-    "@angular/material": "~11.2.3",
-    "@angular/platform-browser": "~11.2.4",
-    "@angular/platform-browser-dynamic": "~11.2.4",
-    "@angular/router": "~11.2.4",
+    "@angular/animations": "~12.2.15",
+    "@angular/cdk": "~12.2.13",
+    "@angular/common": "~12.2.15",
+    "@angular/compiler": "~12.2.15",
+    "@angular/core": "~12.2.15",
+    "@angular/flex-layout": "~12.0.0-beta.35",
+    "@angular/forms": "~12.2.15",
+    "@angular/material": "~12.2.13",
+    "@angular/platform-browser": "~12.2.15",
+    "@angular/platform-browser-dynamic": "~12.2.15",
+    "@angular/router": "~12.2.15",
     "rxjs": "~6.6.6",
     "tslib": "^2.0.0",
-    "zone.js": "~0.10.3"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1102.3",
-    "@angular-eslint/builder": "1.2.0",
-    "@angular-eslint/eslint-plugin": "1.2.0",
-    "@angular-eslint/eslint-plugin-template": "1.2.0",
-    "@angular-eslint/schematics": "1.2.0",
-    "@angular-eslint/template-parser": "1.2.0",
-    "@angular/cli": "~11.2.3",
-    "@angular/compiler-cli": "~11.2.4",
-    "@angular/language-service": "~11.2.4",
+    "@angular-devkit/architect": "^0.1301.3",
+    "@angular-devkit/build-angular": "~12.2.15",
+    "@angular-eslint/builder": "~12.7.0",
+    "@angular-eslint/eslint-plugin": "~12.7.0",
+    "@angular-eslint/eslint-plugin-template": "~12.7.0",
+    "@angular-eslint/schematics": "~12.7.0",
+    "@angular-eslint/template-parser": "~12.7.0",
+    "@angular/cli": "~12.2.15",
+    "@angular/compiler-cli": "~12.2.15",
+    "@angular/language-service": "~12.2.15",
     "@nrwl/cypress": "^12.6.3",
     "@nrwl/workspace": "^11.4.0",
```

Comment:
I was slightly surprised this didn't get updated. This is the (indirect) source of [one of the security alerts](https://github.com/UMM-CSci-3601/3601-iteration-template/security/dependabot/client/package-lock.json/axios/open) that I had looked at for a second and was thinking it would get swept up in all this updating. No worries. We can deal with that later in another round of updates.

Reply:
For the record, `@nrwl/workspace` is up into v13, and the version that fixes the linked security alert is somewhere in the v12 family.

Reply:
Oh, you're right! Yes, it would definitely make sense to update these two. (I may do that in a separate PR.) Thanks! 🙂